assignments/assignment-6/public/styles/styles.css

```diff
@@ -212,3 +212,84 @@ footer {
   color: #fff;
   text-align: center;
 }
+
+.question-window{
+  height: 60%;
+  width: 60%;
+  z-index: 3;
+  position: fixed;
+  top: 20%;
+  left: 20%;
+  background-color: lightblue;
+  display: none;
+}
+
+.question-close{
+  text-align: right;
+  height: 10%
+}
+
+.question-field{
+  height: 15%;
+  text-align: center;
+}
+
+.answer-field{
+  height: 15%;
+  text-align: center;
+  display: none;
+}
+
+.show-answer-container{
+  height: 10%;
+  text-align: center;
+}
+
+.team-button-container{
+  height: 10%;
+  width: 60%;
+  text-align: center;
+  position: fixed;
+  top: 70%;
+}
+
+.answered {
+  text-decoration: line-through;
+}
+
+.team-scores-container-2 {
+  display: inline-block;
+  background-color: yellow;
+}
+
+.team1-score-tracker {
+  display: inline-block;
+}
+
+.team2-score-tracker {
+  display: inline;
```

Comment:
Did you mean `inline-block` here?

Reply:
If we are using all of the same styling on each class, can't we just use one class?

Reply:
With the way the score tracking works, we need to keep track of the score in 2 places. I can unify them under one class and assign point by id as well if you'd prefer that. I was doing it this way because it simplified the JS a bit.

Reply:
no I mean you can have just one styling class that does inline-block for all of them.